docs/logging-usage.rst

```diff
@@ -392,70 +392,105 @@ Delete a sink:
-    >>> from google.cloud.logging.handlers import CloudLoggingHandler, setup_logging
+    >>> from google.cloud.logging.handlers import CloudLoggingHandler
     >>> client = google.cloud.logging.Client()
     >>> handler = CloudLoggingHandler(client)
-    >>> logging.getLogger().setLevel(logging.INFO) # defaults to WARN
-    >>> setup_logging(handler)
-    >>> logging.error('bad news')
+    >>> cloud_logger = logging.getLogger('cloudLogger')
+    >>> cloud_logger.setLevel(logging.INFO) # defaults to WARN
+    >>> cloud_logger.addHandler(handler)
+    >>> cloud_logger.error('bad news')
 
-You can also exclude certain loggers:
+.. note::
+
+    This handler by default uses an asynchronous transport that sends log entries on a background
+    thread. However, the API call will still be made in the same process. For other transport
+    options, see the transports section.
+
+All logs will go to a single custom log, which defaults to "python". The name of the Python
+logger will be included in the structured log entry under the "python_logger" field. You can
+change it by providing a name to the handler:
 
 .. code-block:: python
 
-   >>> setup_logging(handler, excluded_loggers=('werkzeug',)))
+    >>> handler = CloudLoggingHandler(client, name="mycustomlog")
+
+fluentd logging handlers
+==============================
+
+Besides :class:`~google.cloud.logging.handlers.CloudLoggingHandler`, which writes directly to
+the API, two other handlers are provided. :class:`~google.cloud.handlers.GAEHandler.GAEHandler`,
+ which is recommended when running on the Google App Engine Flexible vanilla runtimes, and
+:class:`~google.cloud.handlers.GKE.GKEHandler`, which is recommended when running on Google
+Container Engine with the Stackdriver Logging plugin enabled.
 
+:meth:`~google.cloud.logging.client.get_default_handler` and
+:meth:`~google.cloud.logging.client.setup_logging` will attempt to use
+environment variables to automatically detect whether the code is running in these platforms and use
+the appropriate handler.
 
+In both cases, the fluentd agent is configured to automatically parse log files in an expected
+format and forward them to Stackdriver logging. The handlers provided help set the correct
+metadata such as log level so that logs can be filtered accordingly.
 
-Python logging handler transports
+CloudLoggingHandler transports
 ==================================
 
-The Python logging handler can use different transports. The default is
-:class:`google.cloud.logging.handlers.BackgroundThreadTransport`.
+The :class:`~google.cloud.logging.handlers.CloudLoggingHandler` logging handler can use different
+transports. The default is :class:`google.cloud.logging.handlers.BackgroundThreadTransport`.
 
- 1. :class:`google.cloud.logging.handlers.BackgroundThreadTransport` this is the default. It writes
+ 1. :class:`~google.cloud.logging.handlers.BackgroundThreadTransport` this is the default. It writes
```

Comment:
OK, I think I got it now. That just changes the output in the docs from `google.cloud.logging.handlers.BackgroundThreadTransport` as a link to `BackgroundThreadTransport`.   So yes `~`'s are usually good!